Update demo.md (#34)
eWaterCycle/ewatercycle.github.io: demo.md

You can find the machine at https://demo1.ewatercycle-nle.src.surf-hosted.nl/

> Note that you can use the explorer to browse available models and datasets. To start an experiment on our Jupyterhub you need an account. Please contact us at questions@ewatercycle.org to get one.

Reviewer: ```suggestion
Note that you need an account to start an experiment on our Jupyterhub. Please contact us at questions@ewatercycle.org to get one. Without an account, you can still browse the explorer. 
```

The "note" applies to "needing an account", so imho it's better to keep these subsentences together. Also, this makes it sound like the explorer shows all available models and datasets, but it shows only a subset.
